java-squid/src/main/java/org/sonar/java/se/SymbolicValueFactory.java

```diff
@@ -17,13 +17,11 @@
  * License along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02
  */
-package org.sonar.java;
+package org.sonar.java.se;
 
-import org.junit.runner.RunWith;
-import org.junit.runners.Suite;
-import org.sonar.java.checks.LocksNotUnlockedCheckTest;
+import org.sonar.plugins.java.api.tree.Tree;
 
-@RunWith(Suite.class)
-@Suite.SuiteClasses({LocksNotUnlockedCheckTest.class})
-public class SymExecChecks {
+public interface SymbolicValueFactory {
+
+  SymbolicValue createSymbolicValue(int counter, Tree syntaxNode);
```

Comment:
In the interface the int parameter should be probably called id. It is only a counter in constraint manager for convenience.